python/ray/WebUI.ipynb

```diff
@@ -10,17 +10,266 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
+    "import binascii\n",
     "import os\n",
+    "import pandas as pd\n",
+    "import qgrid \n",
     "import ray\n",
-    "\n",
+    "import redis\n",
+    "import sys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "redis_address = os.environ[\"REDIS_ADDRESS\"]\n",
     "ray.init(redis_address=redis_address)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "addr, port = ray.worker.global_worker.redis_address.split(\":\")\n",
+    "rc = redis.StrictRedis(host=addr, port=port, decode_responses=True, encoding='latin-1', encoding_errors='replace')"
```

Comment:
ideally we wouldn't need to expose the redis client. What about moving the things we use the redis client for into the global state API?